include/error_handler.h

```diff
@@ -89,6 +90,10 @@ class ErrorHandler : public ParameterAcceptor
   /** By default output first table. */
   void output_table(std::ostream &out=std::cout, const unsigned int table_no=0);
 
+  /** By default output first table. The output is according to 
+      to the condition of ConditionalOStream &pout */
+  void conditional_output_table(ConditionalOStream &pout, const unsigned int table_no=0);
```

Comment:
Use the power of `C++`... give this function the same name as the other one... `C++` will know which one to choose...